lib/ClimaLandSimulations/src/utilities/US-Ha1/US-Ha1_parameters.jl

```diff
@@ -0,0 +1,125 @@
+"""Site-specific model parameters for running CliMA LSM on the Harvard Forest
```

Comment:
the specific site files in src/utilities seem like duplicates of the files in src/fluxnet.. are they meant to serve different purposes?

Reply:
No there should only be the files in the fluxnet folder... but I need to figure out how, that's the question I asked Ed above (1st comment of this PR conversation)

Reply:
dont we have three copies though, now? In experiments and then in src/utilities and src/fluxnet?

Reply:
oh i see! that makes sense. should be fixable :)